Gems/Atom/Tools/AtomToolsFramework/Code/Source/DynamicNode/DynamicNodeSlotConfig.cpp

```diff
@@ -31,7 +31,9 @@ namespace AtomToolsFramework
                 ->Field("supportedDataTypeRegex", &DynamicNodeSlotConfig::m_supportedDataTypeRegex)
                 ->Field("defaultDataType", &DynamicNodeSlotConfig::m_defaultDataType)
                 ->Field("defaultValue", &DynamicNodeSlotConfig::m_defaultValue)
-                ->Field("supportsEditingOnNode", &DynamicNodeSlotConfig::m_supportsEditingOnNode)
```

Comment:
Are we losing data with this change? I didn't see a json converter. Or is it OK since Material Canvas is still experimental?

Reply:
No data loss. All of the notes in the test folder were updated to use the new flags. It was only used in a handful of places.